features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Overview.js

```diff
@@ -68,6 +71,14 @@ class Overview extends Component {
         );
     }
 
+    handleRequestClose = () => {
```

Comment:
shall we use the ES6 style class methods instead of class property style methods, because this is still in proposal stage 3 and could be changed in ES7